core/jazz_assets/config/global-config.json

```diff
@@ -31,7 +31,11 @@
 		"cloudwatch_event",
 		"swagger_url",
 		"endpoint_url",
-		"dynamodb_stream_arn"
+    "dynamodb_stream_arn",
+    "dynamodb_role_arn",
+    "sqs_role_arn",
```

Comment:
Not required. It is an iam_role (which we covered already)

Reply:
Done